src/Life/Main/Pull.hs

```diff
@@ -32,7 +32,7 @@ lifePull owner withoutFiles withoutDirs = whatIsLife >>= \case
             _   -> error "Impossible choice"
 
     life :: LifeConfiguration
-    life = LifeConfiguration withoutFiles withoutDirs
+    life = LifeConfiguration withoutFiles withoutDirs (Last $ Just master)
```

Comment:
Maybe it makes sense to create something like:

```haskell
defLifeConfig = LifeConfiguration mempty mempty (Last $ Just master)
```

So later you can just write:

```haskell
life = defLifeConfig
    { lifeConfigurationFiles = withoutFiles
    , lifeConfigurationDirs  = withoutDirs
    }
```

Or something like that. Depends on the strategy about dealing with `branch` field in `LifeConfiguration`.

Reply:
Ok, it looks good.